README.md

```diff
@@ -74,6 +74,11 @@ For 64-bit Linux users,  ```libc6:i386```, ```libstdc++6:i386```, ```libncurses5
 
 __NOTE__: To roll back to the original driver go to: Device Manager -> Right click on device -> Check box for "Delete the driver software for this device" and click Uninstall
 
+### Selecting a SoftDevice
+Softdevices contain the BLE stack and housekeeping, and must be downloaded once before a sketch can be loaded.
+* SoftDevice S110 v8.0.0 supports [Revision](http://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.nrf51%2Fdita%2Fnrf51%2Fcompatibility_matrix%2FnRF51822_ic_revision_overview.html&cp=3_0_1) 2 and 3 of nRF51 in peripheral role. It is 96k in size.
+* SoftDevice S130 v2.0.1 supports Revision 3 of nRF51 in peripheral and central role. It is 108k in size.
+* SoftDevice S132 v2.0.1 supports nRF52 in peripheral and central role. It is 112k in size.
```

Comment:
Soft device base ram use is around 4.x to 5k, but final depends on actual BLE features used. Therefore i don’t see much use in giving an exact amount.